_posts/2020-01-24-proposal-deep-sea-robotics-2

```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: "Proposal: Deep Sea Robotics"
```

Comment:
```suggestion
title: "Proposal: Deep Sea Robotics Part 2"
```

Please also change the name of the file to `2020-01-24-proposal-deep-sea-robotics-part2.md`